src/lib/formatter/SarifFormatter.ts

```diff
@@ -176,7 +177,8 @@ class ESLintSarifFormatter extends SarifFormatter {
 				tool: {
 					driver: {
 						name: engine,
-						version: ESLint.version,
+						// eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
+						version: ESLint.version as string,
```

Comment:
In an upcoming PR, I'm going to be adding `@types/eslint`, and the rule will no longer need to be disabled here. But for now it needs to be disabled because there's no typing information available.